state-chain/pallets/cf-ingress-egress/src/lib.rs

```diff
@@ -1139,6 +1141,7 @@ pub mod pallet {
 			deposit_metadata: CcmDepositMetadata,
 			tx_hash: TransactionHash,
 			deposit_details: Box<<T::TargetChain as Chain>::DepositDetails>,
+			broker_fees: Beneficiaries<T::AccountId>,
```

Comment:
As discussed in the ticket below, and more particularly on the comment, we'll need to pass the accounts and commission/fees for both the broker and the affiliates. Do we use this parameter for both (e.g. first account is the broker) or we separate them?
This also runs into what I recently discussed with @dandanlen  on whether we should make passing the brokerId mandatory for Vault swaps.
https://linear.app/chainflip/issue/PRO-1743/support-for-brokeraffiliate-fees-via-vault-swaps
https://linear.app/chainflip/issue/PRO-1743/support-for-brokeraffiliate-fees-via-vault-swaps#comment-338490a3
I'm not sure there is an easy answer for whether it should be mandatory but it can make sense to separate them here just for flexibility in the future. That's either by having two separate parameters or by this `broker_fees` having two separate parameters.

Reply:
I think this approach is fine for now - we don't need to worry too much about breaking changes here because it only really affects us (it's a witnessed call), and isn't something that is exposed externally.